### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by bugzPDX**
Update ImportErrors to Deprecated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -377,13 +377,10 @@ The -s makes it so that any print statements aren't swallowed if tests pass. The
...
 
-### ImportErrors
+### Deprecated
 
-If you get import errors, make sure the directory `./vendor/src/` is not empty
-or contains only directories that are empty. If they are empty, run the
-following command:
-
-``git submodule update --init --recursive``
+Make sure you don't have a ./vendor or ./vendor-local directory. This functionality has been deprecated. All dependencies
```
Comment (peterbe) [inline README.md]: Put words like that, like "./vendor", in back ticks. I can't demonstrate that here in github because it disappears automatically.

Comment (peterbe) [inline README.md]: Also, can you make the lines no longer than 80 characters each. If you look throughout the rest of the file, you'll notice most lines are capped before 80 characters

File: README.md [hunk 2]
```diff
@@ -377,13 +377,10 @@ The -s makes it so that any print statements aren't swallowed if tests pass. The
 Troubleshooting
 --------------
 
-### ImportErrors
+### Deprecated
```
Comment (peterbe) [inline README.md]: Maybe call this "Deprecated files" instead.

File: README.md [hunk 3]
```diff
@@ -377,13 +377,10 @@ The -s makes it so that any print statements aren't swallowed if tests pass. The
...
 
-If you get import errors, make sure the directory `./vendor/src/` is not empty
-or contains only directories that are empty. If they are empty, run the
-following command:
-
-``git submodule update --init --recursive``
+Make sure you don't have a ./vendor or ./vendor-local directory. This functionality has been deprecated. All dependencies
+are now contained in the requirements.txt file.
```
Comment (peterbe) [inline README.md]: The word "requirements.txt" is also a "computer word" and it should be written in backticks.

**Comment by bugzPDX** [comment]
Noticed your comments after I squashed commits and pushed. I'll make those changes and resubmit.

**Pull request merged**